packages/docs/src/routes/docs/faq/index.mdx

```diff
@@ -209,3 +210,13 @@ An amazing team of contributors around the world living in [Discord](https://qwi
 ## Is Qwik open source?
 
 Yes, [MIT](https://github.com/BuilderIO/qwik/blob/main/LICENSE) and [dependency-free](https://github.com/BuilderIO/qwik/blob/main/packages/qwik/package.json), installing Qwik will not bloat your node_modules nor your lawyers.
+
+# Does Qwik have any downside?
+
+Yes. Every framework, possessing its own strengths and weaknesses, involves a trade off.
+
+1. As a relative newcomer to the JS world, Qwik has yet to develop a community of libraries and best practices.
+2. Qwik’s magic lies in its serialization, but this also brings about some limitations developers wouldn’t otherwise have to consider.
+3. Qwik’s core idea is lazy-loading as much as possible, which sometimes requires developers to restart the dev server to preview changes.
```

Comment:
@manucorporat what do you think of something along these lines

```suggestion
3. Where Qwik outperforms most frameworks is on initial page load. If your use case is an SPA, and any level of server side or static rendering is not a need or interest of yours, Qwik may not offer a huge benefit to you
```

there may be better words - but trying to get at today those who likely _need_ Qwik are "conversion-driven" businesses - aka sites whose initial page load must be lightening fast. For heavy SPAs, Qwik can still have benefits, but marginally less drastic of them. So maybe worth calling out